iam.tf

```diff
@@ -26,7 +26,9 @@ module "iam" {
 
   role = each.value.role
 
-  members       = try(each.value.members, [])
+  members       = try(local.iam[each.value].members, [])
   condition     = try(each.value.condition, null)
   authoritative = try(each.value.authoritative, true)
```

Comment:
this looks odd as we have not converted to using index here.. please fix the map to point to the original index and use index everywhere..